### DISCUSSION TITLE
Grafana - Graph - Time Region - is it possible to have an option to be able to add the time regions created to the Graph Legend ?

### DISCUSSION THREAD

**Discussion by mattsee** [grafana/grafana#38351] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Grafana - Graph - Time Region - is it possible to have an option to be able to add the time regions created to the Graph Legend ?

**Why is this needed**:

Sometimes it is not obvious what the time regions on the graph that are coloured represent.

Having the time regions created and then added to the Graph Legend along with a description of that would make it clearer to people unfamiliar with the graph when looking at it for the first time.

eg. specifiying out of office hours / weekends / maintenance modes etc.

**Comment by hugohaggmark** [comment]
Thanks for taking the time to write this feature request @mattsee

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
